src/index.js

```diff
@@ -1963,6 +1979,9 @@ export class ULabel {
 
     redraw_all_annotations(subtask = null, offset = null, spatial_only = false) {
         // TODO(3d)
+        console.log("")
+        console.log("redraw called")
+        console.log("")
```

Comment:
For logs like this in and the Timing logs, at the very least use [console groups](https://developer.mozilla.org/en-US/docs/Web/API/console/group), and for timing use [console timers](https://developer.mozilla.org/en-US/docs/Web/API/console#timers).

Reply:
I removed these logs from the pr since I just forgot to remove them. But the console groups and console timers look very cool. I didn't know about them.